src/inverseRobot.js

```diff
@@ -1,13 +1,33 @@
 'use strict';
 
-/*
- * @param {object} robot
- *
- * @return {object}
-*/
+const robert = {
+  Robert: 'name',
+  123: 'chipVer',
+  113: 'chipVerPro',
+};
+
+inverseRobot(robert);
 
 function inverseRobot(robot) {
-  // write code here
+  const inversedRobot = {};
+
+  if (robot.hasOwnProperty('name') && robot.name !== 'name') {
+    inversedRobot[robot.name] = 'name';
+  }
+
+  for (const objectKey in robot) {
+    const objectValue = robot[objectKey];
+
+    if (objectKey !== 'name') {
+      if (Object.keys(inversedRobot).includes(objectValue)) {
+        return null;
+      }
+    }
```

Comment:
```suggestion
    if (inversedRobot.hasOwnProperty(value)) {
      return null
    }
```
![image](https://user-images.githubusercontent.com/96409563/223661933-b975317f-a946-4b25-97ee-bcdc3c1b60d8.png)